src/a_star.cpp

```diff
@@ -7,12 +7,11 @@
 
 #include "a_star.hpp"
 
-AStar::AStar() { }
-
-void AStar::setGraph() {
-
+namespace Pathfinding {

```

Comment:
Is there a standard for naming namespaces?

Reply:
There is no standard for namespace names. But every module must have their own namespace. I took the naming conventions of classes for namespaces, since classes are the most like namespaces.